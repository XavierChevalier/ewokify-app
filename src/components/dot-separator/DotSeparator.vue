<template>
  <span class="px-2">â€¢</span>
</template>
