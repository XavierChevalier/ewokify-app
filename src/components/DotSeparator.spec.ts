import { generateStorybookSnapshotTests } from '@/tests/tools/StorybookSnapshot'
import { Default } from './DotSeparator.stories'

describe('DotSeparator', () => {
  generateStorybookSnapshotTests({Default})
})
