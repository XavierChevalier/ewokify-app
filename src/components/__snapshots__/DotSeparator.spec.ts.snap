// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DotSeparator [snapshot] should render the "Default" component correctly 1`] = `
<span
  class="px-1"
>
  â€¢
</span>
`;
